### PR TITLE
migrated all the server actions to trpc participantsRouter

### PR DESCRIPTION
**Pull request by mrkarimoff**
I migrated all the server actions that I created for the 'participant management view' page to trpc participantsRouter.

### CODE REVIEW

File: app/(dashboard)/dashboard/_components/ParticipantsTable/ParticipantsTable.tsx [hunk 1]
```diff
@@ -1,58 +1,70 @@
...
-import { useParticipants } from '../ParticipantsProvider';
-import { ParticipantColumns } from './Columns';
 import ParticipantModal from '../../participants/_components/ParticipantModal';
-import { useState } from 'react';
-import {
-  deleteParticipant,
-  deleteParticipants,
-} from '../../participants/_actions/actions';
-import { type Participant } from '@prisma/client';
+import { ParticipantColumns } from './Columns';
```
Comment (buckhalt) [inline app/(dashboard)/dashboard/_components/ParticipantsTable/ParticipantsTable.tsx:8]: Can you make sure to use import aliases for these and others across this PR?

**Review by buckhalt — CHANGES_REQUESTED** [review]
This looks great! Everything seems to be working well. I'm just requesting the import alias change, but otherwise approve.

File: app/(dashboard)/dashboard/_components/ParticipantsTable/ParticipantsTable.tsx (hunk 1 above)
Comment (mrkarimoff) [reply app/(dashboard)/dashboard/_components/ParticipantsTable/ParticipantsTable.tsx]: Thank you, I changed the import aliases

**Review by buckhalt — APPROVED** [review]
Changes look great.

**Pull request merged**